panic on running snapd

After update to v0.12.0-beta-32-gcde620987342, I'm unable to run snapd after I `make`: 

```
build/bin/snapd -t 0
INFO[0000] Starting snapd (version: v0.12.0-beta-32-gcde620987342)
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x18 pc=0x3401]

goroutine 1 [running]:
main.action(0xc8201440b0)
	/Users/mjbrende/.gvm/pkgsets/go1.5.3/global/src/github.com/intelsdi-x/snap/snapd.go:205 +0x931
github.com/codegangsta/cli.(*App).Run(0xc820152000, 0xc82000a210, 0x3, 0x3, 0x0, 0x0)
	/Users/mjbrende/.gvm/pkgsets/go1.5.3/global/src/github.com/codegangsta/cli/app.go:130 +0xad8
main.main()
	/Users/mjbrende/.gvm/pkgsets/go1.5.3/global/src/github.com/intelsdi-x/snap/snapd.go:190 +0x8bd

goroutine 5 [syscall]:
os/signal.loop()
	/Users/mjbrende/.gvm/gos/go1.5.3/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
	/Users/mjbrende/.gvm/gos/go1.5.3/src/os/signal/signal_unix.go:28 +0x37
```

Notes: 

```
echo $SNAP_PATH
~/.gvm/pkgsets/go1.5.3/global/src/github.com/intelsdi-x/snap/build/
echo $GOPATH
/Users/mjbrende/.gvm/pkgsets/go1.5.3/global
echo $GOROOT
/Users/mjbrende/.gvm/gos/go1.5.3
```

--- a/pkg/globalconfig/flags.go
+++ b/pkg/globalconfig/flags.go
@@ -29,7 +29,7 @@ import (
 )
 
 type config struct {
-	Flags *flagConfig `json:"flags"`
+	Flags flagConfig `json:"flags"`
 }
 
 // FlagConfig struct has all of the snapd flags

